string-manipulation-1/src/is-upper-cased.js

```diff
@@ -0,0 +1,8 @@
+/* exported isUpperCased */
+function isUpperCased(word) {
+  if (word === word.toUpperCase()) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
For this portion, you could just do this: `return word === word.toUpperCase()`

Reply:
Thanks Cody for the feedback!